src/server/config/cmds_test.go

```diff
@@ -85,6 +85,22 @@ func TestActiveContext(t *testing.T) {
 	`))
 }
 
+func TestActiveEnterpriseContext(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+
+	require.YesError(t, run(t, `
+		pachctl config set active-enterprise-context foo 2>&1 | match "context does not exist: foo"
+	`))
+
+	require.NoError(t, run(t, `
+		echo '{}' | pachctl config set context foo --overwrite
+		pachctl config set active-enterprise-context foo
```

Comment:
So anyone who runs this test will have their foo context clobbered?

Reply:
This is a convention in the config CLI tests right now - we could probably change them all to use `UniqueString` in the future.